src/vehicle_base.h

```diff
@@ -771,6 +771,11 @@ struct Vehicle : VehiclePool::PoolItem<&_vehicle_pool>, BaseVehicle, BaseConsist
 
 	inline void SetServiceIntervalIsPercent(bool on) { SB(this->vehicle_flags, VF_SERVINT_IS_PERCENT, 1, on); }
 
+	inline bool AutomaticSeparationIsEnabled() const { return (this->orders.list == nullptr) ? false : this->orders.list->AutomaticSeparationIsEnabled(); }
+	inline void SetAutomaticSeparationIsEnabled(bool enabled) const { if (this->orders.list != nullptr) this->orders.list->SetAutomaticSeparationIsEnabled(enabled); }
```

Comment:
bit more nitpicking, but I would keep the style as above, and a newline extra between `inline` commands.

Reply:
Fixed